src/WebJobs.Script/Binding/ServiceBusScriptBindingProvider.cs

```diff
@@ -17,7 +17,6 @@ namespace Microsoft.Azure.WebJobs.Script
     public class ServiceBusScriptBindingProvider : ScriptBindingProvider
     {
         private readonly string _serviceBusAssemblyName;
-        private EventHubConfiguration _eventHubConfiguration;
```

Comment:
This is a great example of net reduction in code. Most of this code completely disappears because we can infer it from the rule-based EventHub extension.